RecoTracker/LSTCore/src/alpaka/Quintuplet.h

```diff
@@ -0,0 +1,3237 @@
+                                                      float& bridgeInvRadiusMax,
+                                                      float& outerInvRadiusMin,
+                                                      float& outerInvRadiusMax) {
+    float innerInvRadiusErrorBound = 1.9382f;
+    float bridgeInvRadiusErrorBound = 2.2091f;
+    float outerInvRadiusErrorBound = 7.4084f;
+
+    if (innerRadius > 2.0f / (2.f * k2Rinv1GeVf)) {
+      innerInvRadiusErrorBound = 22.5226f;
+      bridgeInvRadiusErrorBound = 21.0966f;
+      outerInvRadiusErrorBound = 19.1252f;
+    }
+
+    innerInvRadiusMax = (1.f + innerInvRadiusErrorBound) / innerRadius;
+    innerInvRadiusMin = alpaka::math::max(acc, 0.f, (1.f - innerInvRadiusErrorBound) / innerRadius);
+
+    bridgeInvRadiusMax = (1.f + bridgeInvRadiusErrorBound) / bridgeRadius;
+    bridgeInvRadiusMin = alpaka::math::max(acc, 0.f, (1.f - bridgeInvRadiusErrorBound) / bridgeRadius);
+
+    outerInvRadiusMax = (1.f + outerInvRadiusErrorBound) / outerRadius;
+    outerInvRadiusMin = alpaka::math::max(acc, 0.f, (1.f - outerInvRadiusErrorBound) / outerRadius);
+
+    return checkIntervalOverlap(alpaka::math::min(acc, innerInvRadiusMin, 1.0 / innerRadiusMax2S),
+                                alpaka::math::max(acc, innerInvRadiusMax, 1.0 / innerRadiusMin2S),
+                                alpaka::math::min(acc, bridgeInvRadiusMin, 1.0 / bridgeRadiusMax2S),
+                                alpaka::math::max(acc, bridgeInvRadiusMax, 1.0 / bridgeRadiusMin2S));
+  };
+
+  template <typename TAcc>
+  ALPAKA_FN_ACC ALPAKA_FN_INLINE void computeSigmasForRegression(TAcc const& acc,
+                                                                 SDL::modules& modulesInGPU,
+                                                                 const uint16_t* lowerModuleIndices,
+                                                                 float* delta1,
+                                                                 float* delta2,
+                                                                 float* slopes,
+                                                                 bool* isFlat,
+                                                                 unsigned int nPoints = 5,
+                                                                 bool anchorHits = true) {
+    /*
+        Bool anchorHits required to deal with a weird edge case wherein 
+        the hits ultimately used in the regression are anchor hits, but the
+        lower modules need not all be Pixel Modules (in case of PS). Similarly,
+        when we compute the chi squared for the non-anchor hits, the "partner module"
+        need not always be a PS strip module, but all non-anchor hits sit on strip 
+        modules.
+        */
+
+    ModuleType moduleType;
+    short moduleSubdet, moduleSide;
+    float inv1 = 0.01f / 0.009f;
+    float inv2 = 0.15f / 0.009f;
+    float inv3 = 2.4f / 0.009f;
```

Comment:
Comment from Matt:
> since these constants appear in more than one place it might be nice if they were put somewhere common

What are these constants? I suppose the lengths of modules? Maybe @YonsiG can clarify?

Reply:
Yes, these are the geometry of modules. They are in units of cm in the code
For 2S module, the stripe is 5cm*90 micro meters.
For PS module pixel, 1.5mm * 100 micro meters
For PS module stripe, 2.4cm * 100 micro meters

Reply:
Thanks! So the first 0.01 is the width of the 2S module? What is the purpose of these seemingly random divisions?

Reply:
No, I think it's the width of PS modules, 100 micro meters in units of cm. Those are getting relative numbers between different length of modules, used for the uncertainty calculation of the tilted modules later. The divisions could come from slopes or relative normalizations between different chi2 components

Reply:
Thank you, I think I got it. It may be useful for you to check the updates, once I am done. I can tag you.